### PR TITLE
People mobile - all Media

### PR DESCRIPTION
**Pull request by Pepko0**
Hi Guys!
I added all media query for People

### CODE REVIEW

File: src/styles/Theme.js [hunk 1]
```diff
@@ -22,5 +22,6 @@ export const theme = {
     tabletVerticalMax: 991,
     tabletHorizontalMax: 1199,
     desktopMini: 1368,
+    mobilePeopleMedium: 452,
```
Comment (Arkadiusz0809) [inline src/styles/Theme.js]: you have to add this feature according to the ascending scale, in the sense that the numbers are in turn

**Review by Arkadiusz0809 — COMMENTED** [review]
Doesn't work properly

File: src/styles/Theme.js (hunk 1 above)
Comment (Pepko0) [reply src/styles/Theme.js]: ok sorry, i will repair it

File: src/features/movieList/components/MovieTile/styled.js [hunk 2]
```diff
@@ -19,8 +19,7 @@ export const MovieTileWrapper = styled.div`
     cursor: pointer;
   }
 
-  @media (max-width: ${({ theme }) =>
-      theme.breakpoints.tabletHorizontalMax}px) {
+  @media (max-width: ${({ theme }) => theme.breakpoints.tabletHorizontalMax}px) {
     margin: 0 auto;
```
Comment (BoosterTech) [inline src/features/movieList/components/MovieTile/styled.js:24]: no format!

**Review by BoosterTech — APPROVED** [review]
Movie button should be in the position exactly between "Movie Browser" and "People" Button but it is a bit to the right. Pictures of people should be getting bigger as screen gets bigger but they are getting away from each other. Looks good in general, Nice job !

**Pull request merged**